spec/requests/lanes_spec.rb

```diff
@@ -26,9 +26,19 @@
                       active_version: true)
   end
 
-  it 'shows the lanes' do
-    get '/workflow_queue/lane_ids?step=dor:accessionWF:shelve'
+  context 'with repository-qualified step names' do
+    it 'shows the lanes' do
+      get '/workflow_queue/lane_ids?step=dor:accessionWF:shelve'
 
-    expect(response.body).to be_equivalent_to '<lanes><lane id="default"/><lane id="fast"/></lanes>'
+      expect(response.body).to be_equivalent_to '<lanes><lane id="default"/><lane id="fast"/></lanes>'
+    end
+  end
+
+  context 'without repository-qualified step names' do
+    it 'shows the lanes' do
+      get '/workflow_queue/lane_ids?step=accessionWF:shelve'
+
+      expect(response.body).to be_equivalent_to '<lanes><lane id="default"/><lane id="fast"/></lanes>'
```

Comment:
missed this one (I know it's short, but it helps conceptually that they are THE SAME.

Reply:
got it!